.github/workflows/ci.yml

```diff
@@ -53,6 +53,41 @@ jobs:
           docker push ${{ env.DOCKER_REGISTRY_URL }}/ucentralgw:$tag
         done
 
+  docker-compose:
+    runs-on: ubuntu-20.04
+    needs: docker
+    if: github.event_name == 'pull_request'
```

Comment:
So what do you think about removing this condition?

Reply:
https://github.com/Telecominfraproject/wlan-cloud-ucentralgw/pull/22#discussion_r660692571 
Waiting for feedback.